### PR TITLE
UI/total client usage

### PR DESCRIPTION
**Pull request by Monkeychip**

### CODE REVIEW

File: ui/app/components/clients/dashboard.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { action } from '@ember/object';
 import { tracked } from '@glimmer/tracking';
 import { format } from 'date-fns';
 
-export default class HistoryComponent extends Component {
+export default class DashboardComponent extends Component {
```
Comment (hellobontempo) [inline ui/app/components/clients/dashboard.js:6]: I noticed the generator labels components `Dashboard` - (without `Component`) not sure if we want to follow with that convention?

File: ui/app/components/clients/total-client-usage.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  { month: 'November', directEntities: 300, nonDirectTokens: 32 },
+  { month: 'December', directEntities: 600, nonDirectTokens: 202 },
+];
+
+// COLOR THEME:
+const BAR_COLOR_DEFAULT = ['#1563FF', '#8AB1FF'];
+
+export default class TotalClientUsage extends Component {
+  @action
+  registerListner(element) {
```
Comment (hellobontempo) [inline ui/app/components/clients/total-client-usage.js:42]: I know this is a WIP - and may even change - but should this be `registerListener`*?

Comment (Monkeychip) [reply ui/app/components/clients/total-client-usage.js:42]: Good catch. I'll amend

File: ui/app/styles/core/charts.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
+.chart-wrapper {
+  border: $light-border;
+  border-radius: $radius-large;
+  padding: $spacing-l $spacing-l $spacing-s $spacing-l;
+  height: 380px;
+  width: 100%;
+
+  display: grid;
```
Comment (hellobontempo) [inline ui/app/styles/core/charts.scss:8]: nice job! thank you for doing this

File: ui/app/templates/components/clients/total-client-usage.hbs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<div class="chart-wrapper">
+  <div class="chart-header">
+    <h2 class="chart-title">{{@title}}</h2>
+    {{#if @description}}
+      <p class="chart-description">{{@description}}</p>
+    {{/if}}
+  </div>
+  <div class="chart-container">
+    <svg
```
Comment (hellobontempo) [inline ui/app/templates/components/clients/total-client-usage.hbs:9]: So grid layout wasn't bothered by SVGs?

**Review by hellobontempo — APPROVED** [review]
thank you! nice job getting this foundation all set up

File: ui/app/templates/components/clients/total-client-usage.hbs (hunk 4 above)
Comment (Monkeychip) [reply ui/app/templates/components/clients/total-client-usage.hbs:9]: Nope! :)

**Pull request merged**